README.md

```diff
@@ -65,4 +65,37 @@ pub fn data_to_json(data: Data) -> json.Json {
 }
 ```
 
+## Usage
+Everything starts with a string representation of your schema written using the [JSON Type Definition Format](https://jsontypedef.com/docs/jtd-in-5-minutes/)
+```gleam
+// you can also achieve this by using the gleam/json.{string} function if you receive the data from stdin or a file
```

Comment:
This comment is incorrect: That function for turning a string into JSON. Remove this comment please.